<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitialSetup</class>
 <widget class="QWidget" name="InitialSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Initial setup</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>490</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>No configuration file has been found. Please customize your application front-end:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_modules_iad">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Inverse Adding-Doubling</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_modules_polarimeter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Polarimeter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_continue">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Modules</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Toolkits</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_toolkits_fluidic">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Fluidic circuit</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_toolkits_light">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Light source</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_connection_usb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>USB</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_connection_ethernet">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ethernet</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Connection</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
